### PR TITLE
add Apollo Engine

### PR DESCRIPTION
**Pull request by Fallion**
- Port on which the server is running is automatically reported by Engine.
- Adds new ENV var for Engine.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,13 +53,14 @@
   },
   "main": "dist/endpoint.js",
   "scripts": {
-    "start": "yarn run nodemon --verbose --ignore __tests__/ --ignore __mocks__/ --exec yarn run babel-node src/index.js",
+    "start":
```
Comment (michalsanger) [inline package.json]: please don't format `.json` with Prettier in this PR

Comment (Fallion) [reply package.json]: Fixed.

**Comment by mrtnzlml** [comment]
@Fallion Could you also please clean up the AWS Lambda tracing? Check the related PR. It's mostly about `graphqlServer.js`...

**Comment by mrtnzlml** [comment]
Or maybe it's not necessary and it can be removed with lambda code later?

**Comment by Fallion** [comment]
@mrtnzlml I'll do it in another PR. I'd like to do small iterations to double check everything works.

**Comment by Fallion** [comment]
@michalsanger Readme updated.

**Pull request merged**